ampersand-select-view.js

```diff
@@ -173,8 +182,20 @@ SelectView.prototype.remove = function () {
     this.el.removeEventListener('change', this.onChange, false);
 };
 
+SelectView.prototype.reset = function() {
+    if(this.startingValue) {
+        this.setValue(this.startingValue, true);
+    } else {
+        this.clear();
+    }
+};
+
 SelectView.prototype.clear = function() {
-    this.setValue('', true);
+    if(this.unselectedText) {
+        this.setValue(undefined, true);
```

Comment:
@nickryall.  My prior suggestion is **not** the correct approach.  It defeats the point of passing in `options: [[true, 'foo'], [false, 'bar']]`.  The value must be coupled to the defined option values.  I will rework my PR.